src/Widgets/DisplayWidget.vala

```diff
@@ -49,8 +49,8 @@ public class Display.DisplayWidget : Gtk.EventBox {
     private Gtk.ComboBox refresh_combobox;
     private Gtk.ListStore refresh_list_store;
 
-    private int real_width = 0;
-    private int real_height = 0;
+    public int real_width = 0;
```

Comment:
```suggestion
    public int real_width { get; private set; default = 0; }
```

We might want to use `private set` since we don't need the ability to set these values publicly.